keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -23,8 +23,8 @@ import com.netflix.spinnaker.keel.api.ec2.cluster.Dependencies
 import com.netflix.spinnaker.keel.api.ec2.cluster.Health
 import com.netflix.spinnaker.keel.api.ec2.cluster.LaunchConfigurationSpec
 import com.netflix.spinnaker.keel.api.ec2.cluster.Location
-import com.netflix.spinnaker.keel.api.ec2.cluster.Moniker
 import com.netflix.spinnaker.keel.api.ec2.cluster.Scaling
+import com.netflix.spinnaker.keel.clouddriver.model.Moniker
```

Comment:
The idea with the package structure was that it would clearly delineate things that were part of the Keel API vs things that were part of the Clouddriver API. Maybe we should move this somewhere common?

Reply:
Sounds good.